### ISSUE TITLE
Issue with screen rotation

### ISSUE THREAD

**Issue by Simgy** [Z-Bolt/OctoScreen#371]
Hello,

I installed last version of Raspberry OS with an hyperpixel 4.0 touch.
I set the hyperpixel and touchscreen of my raspberry in landscape position with : 
- `dtparam=rotate=90,touchscreen-swapped-x-y,touchscreen-inverted-y`

but when I install octoscreen, the window of octoscreen is in portait mode.

I tried lot of solution with no result.
Do you have this type of problem? how did you solve it?
![OctoScreen_portait](https://github.com/Z-Bolt/OctoScreen/assets/18244503/686f369b-303a-4426-abbe-3e35e25fd599)

**Comment by treakifo** [comment]
For what it's worth, I'm having the same problem, also using a Hyperpixel 4.0 touch and config.txt setting. The console text at boot appears in the correct orientation, but OctoScreen needs to be rotated 90 degrees. It was working on the same hardware from an earlier version of the OctoPi OS image, but I stupidly decided to rebuild with the current version.

**Comment by fargodwe** [comment]
Did you try a 180 rotation?

**Comment by treakifo** [comment]
I actually got it working! From Pimoroni's Hyperpixel driver page (https://github.com/pimoroni/hyperpixel4) the section entitled "Totally Manual Rotation", I was able to use the Rotation on the Fly instructions to rotate the screen the the correct value. The command stores the rotation you set and no additional manual config changes are needed.

I tried reproducing the steps on another Raspberry Pi 4 with another Hyperpixel, but I couldn't get it to work again. I finally just cloned the SD card I used to build the first Pi and that finally worked.
